Resources/Public/Javascript/PageView/SearchInDocument.js

```diff
@@ -110,25 +318,22 @@ $(document).ready(function() {
                     resultList += '<li class="noresult">' + $('#tx-dlf-search-in-document-label-noresult').text() + '</li>';
                 }
                 resultList += '</ul>';
-                if (start > 0) {
-                    resultList += '<input type="button" id="tx-dlf-search-in-document-button-previous" class="button-previous" onclick="previousResultPage();" value="' + $('#tx-dlf-search-in-document-label-previous').text() + '" />';
-                }
-                if (data['numFound'] > (start + 20)) {
-                    resultList += '<input type="button" id="tx-dlf-search-in-document-button-next" class="button-next" onclick="nextResultPage();" value="' + $('#tx-dlf-search-in-document-label-next').text() + '" />';
-                }
+                resultList += getNavigationButtons(start, data['numFound']);
                 $('#tx-dlf-search-in-document-results').html(resultList);
             },
             "json"
-        )
-        .done(function( data ) {
+        ).done(function (data) {
             $('#tx-dfgviewer-sru-results-loading').hide();
             $('#tx-dfgviewer-sru-results-clearing').show();
         });
     });
-      // clearing button
-    $('#tx-dlf-search-in-document-clearing').click(function() {
+
+     // clearing button
+     $('#tx-dlf-search-in-document-clearing').click(function() {
         $('#tx-dlf-search-in-document-results ul').remove();
         $('.results-active-indicator').remove();
         $('#tx-dlf-search-in-document-query').val('');
     });
+
+    triggerSearchAfterHitLoad();
```

Comment:
This will be called on the listview, too and will end up in an JavaScript exception

```
Uncaught TypeError: Cannot read property 'split' of undefined
    at getBaseUrl (merged-0478f68d822d724a9d8e6d53edf60223.js?1625082852:4207)
    at triggerSearchAfterHitLoad (merged-0478f68d822d724a9d8e6d53edf60223.js?1625082852:4385)
    at HTMLDocument.<anonymous> (merged-0478f68d822d724a9d8e6d53edf60223.js?1625082852:4466)
```

Reply:
Is SearchInDocument used also in list view?

Reply:
Yes. All JavaScript ist loaded. That's what we changed recently: https://github.com/kitodo/kitodo-presentation/blob/master/Configuration/TypoScript/setup.txt

Reply:
Ok, so I have added check for undefined before trying to split base url.